auctioneerrpc/auctioneer.proto

```diff
@@ -843,7 +861,7 @@ message ServerOrder {
     /*
     The type of the channel that should be opened.
     */
-    uint32 chan_type = 12;
+    OrderChannelType channel_type = 12;
```

Comment:
This was never actually used, so it isn't technically a breaking change right?

Reply:
Correct, and IIUC it also isn't a breaking change according to the proto3 language guide:

> Enumerator constants must be in the range of a 32-bit integer.

> enum is compatible with int32, uint32, int64, and uint64 in terms of wire format (note that values will be truncated if they don't fit). However be aware that client code may treat them differently when the message is deserialized: for example, unrecognized proto3 enum types will be preserved in the message, but how this is represented when the message is deserialized is language-dependent. Int fields always just preserve their value.